index.js

```diff
@@ -549,10 +549,9 @@ Lexer.prototype = {
    */
 
   path: function() {
-    var tok = this.scanEndOfLine(/^ +([^\n]+)/, 'path');
-    // the .trim() is necessary since we want to avoid `  ` from being
-    // matched, as the second space satisfies `[^\n]+`
+    var tok = this.scanEndOfLine(/^ ([^\n]+)/, 'path');
```

Comment:
We have that option for every regex in the file.  I would hope that the JavaScript execution engines would be good enough to optimise it.  If not, and the performance difference is significnat, we should consider writing a babel plugin to optimise it and start using babel to build this module.